pom.xml

```diff
@@ -134,8 +134,7 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>3.14.0</version>
 				<configuration>
-					<source>1.8</source>
-					<target>1.8</target>
+					<release>8</release>
```

Comment:
Side note: This will require JDK >= 9 for building, but I guess it is safe to assume that most users will have that.

(This is not technically needed for the other changes in this pull request.)

Reply:
Requiring JDK 9 is fine. We could even change this to 11. I have no intention to maintain this for older versions.